src/components/post-card/PostCard.jsx

```diff
@@ -74,16 +135,9 @@ const PostCard = ({ post, deleteCard, cardType }) => {
               ))}
           </div>
         )}
-        {/* {cardType === "FOLLOW_CARD" && !checkUserIsFollowed() && (
-          <button
-            onClick={followUser}
-            className="border p-1 px-2 rounded-md hover:bg-hover-color"
-          >
-            Follow
-          </button>
-        )} */}
       </div>
       <div>
+        <p>{media ? "MEDIA" : "NO MEDIA"}</p>
```

Comment:
so, you'll add media later?

Reply:
yes.
currently media feature is not added